nengo_viz/static/viz_netgraph_item.js

```diff
@@ -36,7 +36,7 @@ VIZ.NetGraphItem = function(ng, info) {
     this.minHeight = 5;
 
     this.expanded = false;
-    
+
```

Comment:
It will greatly simplify the merge if we don't include all these whitespace changes.  @tbekolay walked me through a process for trying to get rid of these...